Makefile

```diff
@@ -73,13 +77,36 @@ osx-package: $(OSX_FONT)
 	cp README.md $(OSX_PACKAGE)
 	7z a -tzip -mx=9 $(OSX_PACKAGE).zip ./$(OSX_PACKAGE)
 
+windows-package: $(WINDOWS_FONT)
+	cp $(WINDOWS_FONT) .
```

Comment:
Don't necessarily need a `windows-package` since there's no "package" it's not making a zip/tar.gz.